setup.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from setuptools import setup
 
-packages = ["fsd_utils"]
+packages = ["fsd_utils", "fsd_utils.config"]
```

Comment:
Out of interest, is the idea that each util folder has to be declared as a package here? Can we we just have one package declared here and then import each util as a module in fsd_utils/__init__.py?

Reply:
Ive added automatic package discovery to satiate you.